assignment.hs

```diff
@@ -43,7 +43,7 @@ reciprocal x = . . .
 -- that returns the reciprocal of the square of its parameter
 -- (that it, 1 / x²). Make sure (3) works.
 
--- your code
+oneOverXSquared x = reciprocal (square x)
```

Comment:
We haven't seen this in class, but you can also write

~~~haskell
oveOverXSquared = reciprocal . square
~~~

That creates a new function that is the composition of the originals.